### ISSUE TITLE
Wrong mmap version

### ISSUE THREAD

**Issue by andrewmunro** [Jnnshschl/AmeisenNavigation#2]
Hi, first of all, great project 🥇 

I'm trying to get this running to power pathfinding in a bot I'm building. I've tried downloading the mmaps from the link in the readme, but I get the following error:

`Wrong MMAP version (got:6, expected:12) dtTile 1 1 (mapId: 1)`

I can only assume that download is old? Before I go compiling the trinity src just to get the extractor tools, happen to have an updated download link for ones you are using?

**Comment by andrewmunro** [comment]
Found the latest artifact with the tools pre-compiled here: https://ci.appveyor.com/project/DDuarte/trinitycore/branch/3.3.5/artifacts

**Comment by andrewmunro** [comment]
Hmm, so tried it out with the lastest mmaps and it doesn't work well. The pathfinding tries to navigate it over large hills that are completely impassible. A good test is creating an orc and trying to navigate him to org. He will try walk into hills in the den.

Looking over the changes, trinity have made some efforts to allow NPCs to climb steep hills (by design), something that won't work for bots.

E.G https://github.com/TrinityCore/TrinityCore/pull/24765/commits/de3f79e85bf11aafdb05fcd41bdc9d3965c30f79

I think we need to either:

a) Pin to an older version of the mmap extractor
b) Fork the extractor itself and tweak maxClimb values

**Comment by gr33k** [comment]
I too am running in to this issue. I am trying to use it with the Ameisen Bot though with the same results.

Funny thing is I've tried like 5 different copies to no avail. I even pulled the mmaps from the AzerothCore server I'm running and still not working. I tried both the latest binary here and compiled the latest code on my own with similar results. The compiled version seems to give a slightly different output (Debug/verbose perhaps?) but still the same error about the map version:
![image](https://user-images.githubusercontent.com/5171932/98517520-e3bd5280-2222-11eb-9ad7-df36f8861777.png)

**Comment by Jnnshschl** [comment]
Ye, sorry for my late response, I've modified the MMAP extractor and tweaked the navmesh params for my bot. I need to write a complete guide for it, because the protocol has changed a lot lately. I'll try to do that within the next few days.

**Comment by andrewmunro** [comment]
@Jnnshschl Yes we noticed the same thing when looking into it, they've made a bunch of changes recently to make the navmeshes more "blizzlike".

TLDR: NPC navmeshes are very different from what a bot navmesh should look like, so a custom extractor is required. I'd be interested to see where you get with it. We ended up forking the MMAP extractor too and tweaking some vars, but nothing worth sharing.
